client/index.js

```diff
@@ -1,20 +1,5 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-
-    };
-  }
-
-  render() {
-    return (
-      <div>App rendering!</div>
-    );
-  }
-}
+import App from './components/App.jsx';
```

Comment:
@drtcxrch - Yeah, looks like we ran into the same problem in setting up these tests haha. Also looks like we solved it the same way!